addons/web/__manifest__.py

```diff
@@ -562,6 +566,10 @@
 
             ('include', 'web.frontend_legacy'),
             'web/static/src/legacy/frontend/*/tests/*.js',
+
+            # Old notification service kept for legacy tests compatiblity
+            'web/static/src/legacy/js/services/notification_service.js',
+            'web/static/src/legacy/js/widgets/notification.js',
```

Comment:
actually it is already in web.frontend_legacy. What is missing is the legacy test file, which should be moved to web.frontend_legacy_tests